WalletWasabi.Gui/Tabs/WalletManager/LoadWalletViewModel.cs

```diff
@@ -440,122 +394,109 @@ public async Task<KeyManager> LoadKeyManagerAsync(bool requirePassword, bool isH
 				var walletName = selectedWallet.WalletName;
 				if (isHardwareWallet)
 				{
-					if (selectedWallet.HardwareWalletInfo is null)
+					using (var cts = new CancellationTokenSource(TimeSpan.FromMinutes(3)))
```

Comment:
This `cts` will expire in 3 minutes but there is one inner `ctsSetup` that expires in 21 minutes. I think this is wrong.

Reply:
Correct, the first cts was used first, but then I realized that setup response may take a very long time, because they have to write down words and stuff, so I added another one and so the first one wasn't needed there anymore.
Anyway, fixed it.